astroquery/jplhorizons/tests/test_jplhorizons.py

```diff
@@ -256,22 +256,23 @@ def test_vectors_query_payload():
     res = jplhorizons.Horizons(id='Ceres', location='500@10',
                                epochs=2451544.5).vectors(
                                    get_query_payload=True)
-
     assert res == OrderedDict([
-        ('format', 'text'),
```

Comment:
Please refrain from reformatting like this. Add the new dict element, but leave the rest alone.

Reply:
Switching the order back will cause the test to fail, because it is an equality test against an `OrderedDict`, and changes I need to make in order to construct the payload to include target coordinates alter the order in which keys are inserted in the `OrderedDict`.

Reply:
will add a changelog entry and update narrative docs tomorrow or tuesday.

Reply:
I don't mean the order, but don't do reformatting, no need to change 15 lines when only one or two need to be shuffled around

Reply:
oh you mean the indentation. sorry, didn't even notice.